docs/docs-content/clusters/edge/edge-configuration/installer-reference.md

```diff
@@ -377,6 +377,7 @@ installation is complete.
 | Parameter                                      | Description                                                                                                                                                      | Default |
 | ---------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
 | `install.bind_mounts`                          | The list of folders to bind mount from the installer to the Edge host                                                                                            | None    |
+| `install.bind_mounts.extra-dirs-rootfs`        | The list of folders to bind mount from the installer to the Edge host's root file system. These folders are created in the `rootfs` during installation.         | None    |
```

Comment:
```suggestion
| `install.bind_mounts.extra-dirs-rootfs`        | The list of folders to bind mount from the installer to the Edge host's root file system. These folders are created in the `rootfs` filesystem during installation.         | None    |
```

Reply:
It's minor, but should we add this after `rootfs`? Or is it more accurate to say directory or even filesystem?